index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="index.css">
+        <script src="https://kit.fontawesome.com/5031df2cf9.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <header>
+          <nav>
+            <div class="logo">
+                My Logo
+            </div>
```

Comment:
- [ ] Please use a a link (a tag) for logo here. This is for three reasons: convention, semantics, and user experience.